[IMP] l10n_fr: new EDI reports (#18243)
odoo/documentation: content/applications/finance/fiscal_localizations/france.rst

     select :guilabel:`Download XML` to generate an XML file.
   - Specific tags, related to the DAS2 Report, indicate the nature of payments, including
     :guilabel:`Fees (H)`, :guilabel:`Commissions (C)`, :guilabel:`Brokerage (CO)`,
     :guilabel:`Rebates (R)`, :guilabel:`Attendance fees (JP)`, :guilabel:`Copyright (DA)`,
     :guilabel:`Inventor's rights (DI)`, and :guilabel:`Other remuneration (AR)`

.. seealso::
   - `Guide utilisateur français pour la transmission des informations relatives à la DAS2
     Honoraires <https://edificas.org/download/document/ab4310d4-3bba-400e-87fe-a6e75f96e674/zip>`_
   - `Formulaire n°DAS2 - Etat des honoraires, vacations, commissions, courtages, ristournes et
     jetons <https://www.impots.gouv.fr/formulaire/das2/etat-des-honoraires-vacations-commissions-courtages-ristournes-et-jetons>`_

.. _localizations/france/das2-contact:

Contact
*******

To provide the necessary contact information, include the following details:

- :guilabel:`Name`: Provide the full name for individuals or the business name.
- :guilabel:`Address`: Include the :guilabel:`Street`, :guilabel:`ZIP` code, :guilabel:`City`, and
  :guilabel:`Country`.

> - :guilabel:`SIRET` number: In the :guilabel:`VAT` field, click the :icon:`fa-plus`

Reviewer: I'll check this in a few minutes but we might be storing the SIRET number in the company registry field not the new additional info field

Edit: Yuupp `not company.company_registry, self.env._("company SIRET is not set")`
I'll probably change this to use the new field, I'll let you know when I do!

  :guilabel:`(plus)` icon and select :guilabel:`SIRET` to enter the :guilabel:`SIRET` number.
- :guilabel:`DAS2 - Profession`: In the :guilabel:`Accounting tab`, fill in the :guilabel:`DAS2 -
  Profession` field located in the :guilabel:`General` section.

.. note::

>    The partner's birth date is required for DAS2 beneficiaries who are EU residents but not French.

Reviewer: AFAIK this isn't about being French but more about residing outside France
> la ligne DN (date de naissance) est à compléter obligatoirement pour les personnes physiques domiciliées en Union européenne (hors France) et ayant perçu des droits d’auteurs (DA) ou des droits d’inventeur (DI) (Bulletin officiel des Finances Publiques – Impôts [BOI-BNC-SECT-20-10-60, § 40]) ;

.. _localizations/france/das2-submit:

Submit
******

.. note::

>    - A minimum payment of €2,400 is necessary for the corresponding line to be included in the DAS2

Reviewer: Maybe also mention that it can be multiple small payments adding up to 2.4k 
Maybe "A minimum total payment" or something similar

     report.
   - To successfully send the EDI DAS2 report, the :guilabel:`APE` and the company's main
     :guilabel:`Activity` fields must be completed on the :doc:`company record
     </applications/general/companies>`.

To submit the EDI DAS2 report, follow these steps:

> #. In the :guilabel:`DAS2 Report` view, click the :icon:`fa-cog` (:guilabel:`gear`) icon, and select

Reviewer: ~~Checked in master, there's no gear icon
So the only way to get the XML doc is to follow the Returns flow and download it from the asynchronous docs view~~

False alarm, I was on the US company that's why the gear didn't show up
Maybe we should mention something for that?

Author: Users mostly use their company database, so we usually don't specify which country's database they need to follow the instructions for.

   #. Click the :guilabel:`DAS2 Report` line, then click the :guilabel:`DAS2 Report (FR)` button to
      access and review it.
   #. After reviewing, click :guilabel:`Tax Return` in the breadcrumbs to return to the
      :guilabel:`DAS2 Report` item.
   #. Click :guilabel:`Validate` to complete the :guilabel:`Review` step; the DAS2 report PDF is
      then available in the chatter.
   #. Click :guilabel:`Submit` to submit the DAS2 Report to |DGFiP|.
   #. In the :guilabel:`DAS2 Report` window, update the :guilabel:`Point of Contact` field with the
      name of the person responsible for sending the report directly to |DGFiP|, and click
      :guilabel:`Send DAS2 Report`.

.. _localizations/france/reporting-fiscal-declaration:

Simplified fiscal declaration
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Most taxpayers are required to file a simplified fiscal declaration as part of the standard
procedure.

To access the declaration, go to :menuselection:`Accounting --> Reporting --> Fiscal Declaration`.
The Fiscal Declaration includes various sections, such as :guilabel:`2033 - A` and :guilabel:`2031
- Annexes`. Use the section buttons to navigate through the report.

.. tip::

>    - To update values, click the :icon:`fa-pencil` :guilabel:`(pencil)` icons; changes are saved

Reviewer 2: Maybe we could do a tip for the lines that need information about a partner, for example the accountant name and address, we simplified the encoding by allowing to put a partner instead. So they could encounter input field where they need to put a partner

Reporting
---------

To have access to these accounting reports specific to France, :doc:`Install
</applications/general/apps_modules>` the :guilabel:`France - Accounting` (`l10n_fr_account`):

- :guilabel:`Bilan comptable (FR)` (:guilabel:`Balance Sheet`)
- :guilabel:`Compte de résultats (FR)` (:guilabel:`Profit and Loss`)
- :guilabel:`Rapport de taxes (FR)` (:guilabel:`Tax Report`)
- :ref:`DAS2 report <localizations/france/reporting-das2>`
- :ref:`Simplified Fiscal Declaration <localizations/france/reporting-fiscal-declaration>`

.. _localizations/france/reporting-das2:

DAS2 report
~~~~~~~~~~~

.. note::
   Make sure to :doc:`install </applications/general/apps_modules>` the :guilabel:`France -

>    Accounting Reports` (`l10n_fr_reports`) module.

Reviewer 3: Installing the _France - Accounting_ module installs the _France - Accounting Reports_ one, so it may not be necessary to specify this?

Any individual or legal entity that pays commissions, brokerage fees, rebates, gratuities, or other
similar compensation to third parties during their professional activities must file a fee

> declaration (DAS2 Report).

Reviewer 3: ```suggestion
declaration (DAS2 report).
```

To access the DAS2 report, go to :menuselection:`Accounting --> Reporting --> DAS2 Report`.

.. tip::

>    Specific tags, related to the DAS2 Report, indicate the nature of payments, including

Reviewer 3: ```suggestion
   Specific tags, related to the DAS2 report, indicate the nature of payments, including
```

   :guilabel:`Fees (H)`, :guilabel:`Commissions (C)`, :guilabel:`Brokerage (CO)`, :guilabel:`Rebates
   (R)`, :guilabel:`Attendance fees (JP)`, :guilabel:`Copyright (DA)`, :guilabel:`Inventor's rights

>    (DI)`, and :guilabel:`Other remuneration (AR)`

Reviewer 3: ```suggestion
   (DI)`, and :guilabel:`Other remuneration (AR)`.
```

- :guilabel:`SIRET` number: In the :guilabel:`VAT` field, click the :icon:`fa-plus`
  :guilabel:`(plus)` icon and select :guilabel:`SIRET` to enter the :guilabel:`SIRET` number.
- :guilabel:`DAS2 - Profession`: In the :guilabel:`Accounting tab`, fill in the :guilabel:`DAS2 -
  Profession` field located in the :guilabel:`General` section.

.. note::
   The partner's birth date is required for DAS2 beneficiaries who are EU residents (excluding
   France) and received copyright royalties (DA) or inventor's fees (DI).

.. _localizations/france/das2-submit:

Submit
******

.. note::
   - A minimum total payment of €2,400 is required for the corresponding line to be included in the
     DAS2 report.
   - To successfully send the EDI DAS2 report, the :guilabel:`APE` and the company's main
     :guilabel:`Activity` fields must be completed on the :doc:`company record
     </applications/general/companies>`.

To submit the EDI DAS2 report, follow these steps:

#. In the :guilabel:`DAS2 Report` view, click the :icon:`fa-cog` (:guilabel:`gear`) icon, and select
   :guilabel:`EDI DAS2`.
#. In the :guilabel:`DAS2 Report Generation` window, update the :guilabel:`Point of Contact` field

>    if necessary, and select the contact responsible for directly sending the report to |DGFiP|.

Reviewer 3: ```suggestion
#. In the :guilabel:`DAS2 Report Generation` window, update the :guilabel:`Point of Contact` field
   if necessary by selecting the contact responsible for directly sending the report to |DGFiP|.
```

#. Click :guilabel:`Send DAS2 Report`.

The DAS2 report includes the total amount of the sums paid, including VAT if beneficiaries are

> subject to VAT.

Reviewer 3: Why mention this at the end - technically, after submission?

> .. tip::

Reviewer 3: If you know why one could use this alternative solution, it could be worth mentioning it :)

> 

Reviewer 3: ```suggestion
```

        completed.
      - A :guilabel:`DAS2 Report (FR)` button to open the report.
      - A :icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis)` menu for additional options.

   #. Click the :guilabel:`DAS2 Report` line, then click the :guilabel:`DAS2 Report (FR)` button to
      access and review it.
   #. After reviewing, click :guilabel:`Tax Return` in the breadcrumbs to return to the
      :guilabel:`DAS2 Report` item.
   #. Click :guilabel:`Validate` to complete the :guilabel:`Review` step; the DAS2 report PDF is
      then available in the chatter.
   #. Click :guilabel:`Submit` to submit the DAS2 Report to |DGFiP|.
   #. In the :guilabel:`DAS2 Report` window, update the :guilabel:`Point of Contact` field if
      necessary, select the contact responsible for directly sending the report to |DGFiP|, and
      click :guilabel:`Send DAS2 Report`.

.. _localizations/france/reporting-fiscal-declaration:

Simplified fiscal declaration
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Most taxpayers are required to file a simplified fiscal declaration as part of the standard
procedure.

To access the declaration, go to :menuselection:`Accounting --> Reporting --> Fiscal Declaration`.

> The Fiscal Declaration includes various sections, such as :guilabel:`2033 - A` and :guilabel:`2031

Reviewer 3: ```suggestion
The fiscal declaration includes various sections, such as :guilabel:`2033 - A` and :guilabel:`2031
```

- Annexes`. Use the section buttons to navigate through the report.

.. tip::
   - To update values, click the :icon:`fa-pencil` :guilabel:`(pencil)` icons; changes are saved
     automatically.
   - For lines requiring information, such as an accountant's name or an address, click the
     :icon:`fa-caret-down` :guilabel:`(down arrow)` icon to select the relevant
     :doc:`contact <../../essentials/contacts>`.
   - Where available, click :guilabel:`Add` or :guilabel:`Add new section` lines to insert lines or
     sub-sections within the report.

>    - To generate an XML file, click the :icon:`fa-cog` (:guilabel:`gear`) icon and select

Reviewer 3: ```suggestion
   - To generate an XML file, click the :icon:`fa-cog` :guilabel:`(gear)` icon and select
```
For consistency ;)

     :guilabel:`Download XML`.

.. _localizations/france/fiscal-declaration-submit:

Submit
******

> To submit the Fiscal Declaration directly to |DGFiP| through Odoo, follow these steps:

Reviewer 3: ```suggestion
To submit the fiscal declaration directly to |DGFiP| through Odoo, follow these steps:
```

#. Go to :menuselection:`Accounting --> Reporting --> Fiscal Declaration`. In the :guilabel:`Fiscal

>    Declaration` view, click :guilabel:`Returns`.

Reviewer 3: ```suggestion
#. Go to :menuselection:`Accounting --> Reporting --> Fiscal Declaration` and click :guilabel:`Returns`.
```
Could you somehow end up on another view?

> #. Remove the :guilabel:`To do` filter in the Tax Return list view to display the :guilabel:`Fiscal

Reviewer 3: ```suggestion
#. Remove the :guilabel:`To Do` filter to display the :guilabel:`Fiscal
```
I'm not sure it's necessary to specify which view you're on every time, unless another view could somehow appear?

   Declaration` line.
#. The :guilabel:`Fiscal Declaration` line includes:

   - A period (year).
   - A deadline date.
   - The related company and :ref:`branch(es) <general/branches>`, if applicable.
   - Action steps, such as :guilabel:`Review` and :guilabel:`Submit`, which turn green when
     completed.

>    - A :guilabel:`Fiscal Declaration` button to open the :guilabel:`Fiscal Declaration`.

Reviewer 3: ```suggestion
   - A :guilabel:`Fiscal Declaration` button to open the fiscal declaration.
```
